libpod/util.go

```diff
@@ -76,3 +79,29 @@ func GetPolicyContext(path string) (*signature.PolicyContext, error) {
 	}
 	return signature.NewPolicyContext(policy)
 }
+
+// RemoveScientificNotationFromFloat returns a float without any
+// scientific notation if the number has any.
+// golang does not handle conversion of float64s that have scientific
+// notation in them and otherwise stinks.  please replace this if you have
+// a better implementation.
+func RemoveScientificNotationFromFloat(x float64) (float64, error) {
+	bigNum := strconv.FormatFloat(x, 'g', -1, 64)
+	breakPoint := strings.IndexAny(bigNum, "Ee")
+	if breakPoint > 0 {
+		bigNum = bigNum[:breakPoint]
+	}
+	result, err := strconv.ParseFloat(bigNum, 64)
+	if err != nil {
+		return x, errors.Wrapf(err, "unable to remove scientific number from calculations")
+	}
+	return result, nil
+}
+
+// ClearScreen clears the terminal (aka "clear") and sets the
+// cursor position to the upper left
+func ClearScreen() {
```

Comment:
I think this is more a cmd/kpod thing. We have cmd/kpod/common.go but it's getting huge... maybe start a cmd/kpod/util.go?

Reply:
moved inline